pkg/kubelet/eviction/defaults_windows.go

```diff
@@ -0,0 +1,27 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package eviction
+
+// DefaultEvictionHard includes default options for hard eviction.
+var DefaultEvictionHard = map[string]string{
+	"memory.available":  "500Mi",
+	"nodefs.available":  "10%",
+	"imagefs.available": "15%",
```

Comment:
Do we need or want these?  Will these affect how storagge evicts?

Reply:
The nodefs and imagefs are already the defaults for Windows.
Initially I had left them out when I made new Windows defaults but someone said I should leave them in (let me try and find the discussion)

Reply:
https://github.com/kubernetes/kubernetes/pull/122922#discussion_r1462516048